src/js/modules/core/components/MastheadBar.js

```diff
@@ -150,7 +161,7 @@ const MastheadBar = ({
         </div>
       </div>
       <div className={classes.sectionsBarContainer}>
-        <FeaturedSectionsBar />
+        <FeaturedSectionsBar omitSearch={true} />
```

Comment:
omitSearch?

Reply:
I want this bar to show a search bar in Masthead, but not MastheadBar, because MastheadBar already has a search button. An easy fix would be to remove the search button from the MastheadBar, but aesthetics.

Reply:
Reconsider naming, necessity, & purpose.